docs/notes/migration_guide.rst

```diff
@@ -145,7 +143,10 @@ Step 5: Migrate public headers
       * - ``fastdds/rtps/common/Time_t.hpp in namespace{fastdds}``
         - ``fastdds/dds/core/Time_t.hpp in namespace{fastdds::dds}``
 
-   Also, the ``fixed_size_string.hpp`` implementation has been migrated from Fast DDS package to Fast CDR.
+   Also, the ``fixed_size_string.hpp`` implementation has been migrated from ``fastrtps/utils/fixed_size_string.hpp``
+   to ``fastcdr/cdr/fixed_size_string.hpp``.
+
+   .. TODO:: Fix this table, ``fastdds/rtps/DomainParticipantQos.hpp`` has wrong path and doesn't include the content.
```

Comment:
This file is not in this path, and doesn't match the content. I think maybe DomainParticipantExtendedQos.hpp?